Makefile

```diff
@@ -419,3 +420,159 @@ image-mnist-job-test-push: image-mnist-job-test-build ## Push container image wi
+	kubectl wait --for=condition=Ready pod/$$ODH_POD_NAME -n openshift-operators
+	kubectl apply -f contrib/configuration/odh/default-dsci.yaml --server-side
+	kubectl apply -f contrib/configuration/odh/default-dsc.yaml --server-side
+
+.PHONY: delete-service-mesh-operator
+delete-service-mesh-operator: ## Delete Service Mesh Operator
+	@echo -e "\n==> Deleting Service Mesh Operator \n"
+	kubectl delete subscription servicemeshoperator -n openshift-operators
+	-export CLUSTER_SERVICE_VERSION=`kubectl get clusterserviceversion -n openshift-operators -l operators.coreos.com/servicemeshoperator.openshift-operators -o custom-columns=:metadata.name`; \
+	kubectl delete clusterserviceversion $$CLUSTER_SERVICE_VERSION -n openshift-operators
+
+.PHONY: install-service-mesh-operator
+install-service-mesh-operator: ## Install Service Mesh Operator
+	@echo -e "\n==> Installing OpenShift Service Mesh Operator"
+	kubectl create -f contrib/configuration/service-mesh-operator-subscription.yaml
+	kubectl wait -n openshift-operators subscription/servicemeshoperator --for=jsonpath='{.status.state}'=AtLatestKnown --timeout=180s
+	@while [[ -z $$(kubectl get deployment/istio-operator -n openshift-operators) ]]; do echo "."; sleep 10; done
+	kubectl wait --for=condition=available deployment/istio-operator -n openshift-operators --timeout=180s
+
+##@ GPU Support
+.PHONY: install-nfd-operator
+install-nfd-operator: ## Install NFD operator ( Node Feature Discovery )
+	@echo -e "\n==> Installing NFD Operator \n"
+	-kubectl create ns openshift-nfd
+	kubectl create -f contrib/configuration/nfd-operator-subscription.yaml
+	@echo -e "\n==> Creating default NodeFeatureDiscovery CR \n"
+	@while [[ -z $$(kubectl get customresourcedefinition nodefeaturediscoveries.nfd.openshift.io) ]]; do echo "."; sleep 10; done
+	@while [[ -z $$(kubectl get csv -n openshift-nfd --selector operators.coreos.com/nfd.openshift-nfd) ]]; do echo "."; sleep 10; done
+	kubectl get csv -n openshift-nfd --selector operators.coreos.com/nfd.openshift-nfd -ojsonpath={.items[0].metadata.annotations.alm-examples} | jq '.[] | select(.kind=="NodeFeatureDiscovery")' | kubectl apply -f - --validate=false
+
+.PHONY: delete-nfd-operator
+delete-nfd-operator: ## Delete NFD operator
+	@echo -e "\n==> Deleting NodeFeatureDiscovery CR \n"
+	kubectl delete NodeFeatureDiscovery --all -n openshift-nfd
+	@while [[ -n $$(kubectl get NodeFeatureDiscovery -n openshift-nfd) ]]; do echo "."; sleep 10; done
+	@echo -e "\n==> Deleting NFD Operator \n"
+	-kubectl delete subscription nfd -n openshift-nfd
+	-export CLUSTER_SERVICE_VERSION=`kubectl get clusterserviceversion -n openshift-nfd -l operators.coreos.com/nfd.openshift-nfd -o custom-columns=:metadata.name`; \
+	kubectl delete clusterserviceversion $$CLUSTER_SERVICE_VERSION -n openshift-nfd
+	-kubectl delete ns openshift-nfd
+
+.PHONY: install-nvidia-operator
+install-nvidia-operator: ## Install nvidia operator
```

Comment:
GPUs are not discovered on workbench creation, I think it's because we need to create the `AcceleratorProfile`. Something like this works:
```
apiVersion: dashboard.opendatahub.io/v1
kind: AcceleratorProfile
metadata:
  name: gpu-accelerator-profile
  namespace: default
spec: {
  displayName: gpu,
  enabled: true,
  identifier: nvidia.com/gpu
}
```

Reply:
Did you encounter this on ODH or RHOAI or both?

Reply:
Tested only RHOAI